pycbc/inference/models/gaussian_noise.py

```diff
@@ -1201,7 +1201,8 @@ def create_waveform_generator(
     except KeyError:
         raise ValueError("no approximant provided in the static args")
 
-    generator_function = generator_class.select_rframe_generator(approximant)
+    domain = static_params.get('preferred_domain', 'none')
```

Comment:
Any reason why you're using the string 'none' instead of the type `None`? The latter will be a bit better to work with, to avoid spelling errors.